pkg/azureclients/storageaccountclient/interface.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// APIVersion is the API version for network.
-	APIVersion = "2019-06-01"
+	APIVersion = "2021-02-01"
```

Comment:
has this api version available in all clouds?

Reply:
Yes it is as per this doc : https://docs.microsoft.com/en-us/rest/api/storagerp/